### PR TITLE
Add Reimbursements email action

### PR DESCRIPTION
**Pull request by casassg**
- Adds new action for travel reimbursement emails

### CODE REVIEW

**Review by quim-motger — APPROVED** [review]
Changes reviewed. 
TR email send working OK.

File: register/models.py [hunk 1]
```diff
@@ -8,11 +8,16 @@
...
 
-
+# Votes weight
 TECH_WEIGHT = 0.2
-
 PERSONAL_WEIGHT = 0.8
 
+# Reimbursement tiers
+COUNTRIES_REIMBURSEMENT = [
+    (['Spain', ], 50),
```
Comment (ani300) [inline register/models.py]: Maybe the comma after Spain is not necessary?

**Review by ani300 — COMMENTED** [review]
Apart from that small thing, LGTM

File: register/models.py (hunk 1 above)
Comment (casassg) [reply register/models.py]: Moved this to a CSV for an easier maintenance

**Review by ani300 — APPROVED** [review]

**Pull request merged**